src/scss/chatItem.scss

```diff
@@ -0,0 +1,65 @@
+$bubbleColor: #E2E2E2;
+
+.chat-item{
+
+    font-size: 20px;
+    width: fit-content;
+    max-width: 50%;
+    height: fit-content;
+    margin: 10px;
+    padding: 24px;
+    border-radius: 2rem;
+
+    position: relative;
+    
+    flex-direction: column;
+
+    background-color: $bubbleColor;
+
+    animation-duration: 100ms;
+    transition: all;
+}
+
+    //Optional
+.friend {
+    //add animation from left to target
+}
+
+.user {
+    //add animation from right to target
+}
+
+.friend > .user-picture {
+    background-color: #FFA3A3;
+}
+
+.user-picture {
+    width: 24px;
+    height: 24px;
+    border-radius: 6px !important;
+    background-color: #A8D5FF;
+    margin: 0 10px ;
+    object-fit: cover;
+    display: flex;
+}
+
+.user-picture > span {
+    margin: 0 3px;
+}
+
+span{
+    width: fit-content;
+}
+
+.user > div {
+    margin-left: auto;
+    border-radius: 2rem 0 2rem 2rem;
+}
+
+.user > .user-picture {
+   flex-direction: row-reverse;
+}
+
+.friend > div {
+    border-radius: 0 2rem 2rem 2rem;
```

Comment:
to get closer to the figma design consider this on line 64 instead: 
border-radius: 0 1rem 1rem 1rem;